apps/server/src/modules/oauth/service/oauth.service.ts

```diff
@@ -98,18 +99,24 @@ export class OAuthService {
 			issuer: oauthConfig.issuer,
 			audience: oauthConfig.clientId,
 		});
-		if (typeof verifiedJWT === 'string' || verifiedJWT instanceof String)
+		if (typeof verifiedJWT === 'string' || verifiedJWT instanceof String) {
 			throw new OAuthSSOError('Failed to validate idToken', 'sso_token_verfication_error');
+		}
```

Comment:
Why is it typeof string?

Reply:
removed the typeof part. instanceof suffices here